examples/webpack-demo-vanilla-bundle/src/examples/example07.ts

```diff
@@ -315,6 +315,73 @@ export class Example7 {
     this.sgb.filterService.clearFilters();
   }
 
+  
+  allFilters() {
+    const grid = this.sgb;
+    $(`<div id="modal-allFilter" class="modal is-active" >
```

Comment:
hey I just saw this while relooking at this new code sample, and that worries me a bit. I know it's for demo purpose but I'm not a fan of using that kind of html mostly because it's not sanitized and you're using jQuery to do it. You can maybe change it to something like 
```ts
import * as DOMPurify from 'dompurify';

const modalHtml =`<div id="modal-allFilter" class="modal is-active" >...</div>`;
document.body.appendChild(DOMPurify.sanitize(modalHtml));
```
this would be better since it's both sanitized and isn't using jQuery. We could also create a new package in the future for this, similar to what I've done with [Composite Editor](https://github.com/ghiscoding/slickgrid-universal/tree/master/packages/composite-editor-component) and use similar CSS Classes so that it works the same in all framework, it would also be in plain JS code like the Composite Editor one (but probably more as a future idea)

Reply:
Ready! 
I added the DOMPurify.